docker/1.4.8/README.md

```diff
@@ -42,7 +42,7 @@ docker build -t mosquitto:1.4.8 .
 ##Run
 Run a container using the new image:
 ```
-docker run -it -p 1883:1883 -p 9001:9001 -v mosquitto.conf:/mosquitto/data/mosquitto.conf -v /mosquitto/data -v /mosquitto/log mosquitto:1.4.8
+docker run -it -p 1883:1883 -p 9001:9001 -v `pwd`/mosquitto.conf:/mosquitto/config/mosquitto.conf -v /mosquitto/data -v /mosquitto/log mosquitto:1.4.8
```

Comment:
You are right ⚡️  I don't know why I assumed that `-v <path>:<path>` is equal `-v <path>`.
I would use `<path-to-configuration-file>` also in this example:
```
docker run -it -p 1883:1883 -p 9001:9001 -v <path-to-configuration-file>:/mosquitto/config/mosquitto.conf -v /mosquitto/data -v /mosquitto/log mosquitto:1.4.8
```